specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json

```diff
@@ -0,0 +1,430 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of data policy manifests.",
+            "schema": {
+              "$ref": "#/definitions/DataPolicyManifestListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CloudError": {
+      "x-ms-external": true,
+      "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse",
+      "description": "An error response from a policy operation."
+    },
+    "DataManifestCustomResourceFunctionDefinition": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The function name as it will appear in the policy rule. eg - 'vault'."
+        },
+        "fullyQualifiedResourceType": {
+          "type": "string",
+          "description": "The fully qualified control plane resource type that this function represents. eg - 'Microsoft.KeyVault/vaults'."
+        },
+        "defaultProperties": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The top-level properties that can be selected on the function's output. eg - [ \"name\", \"location\" ] if vault().name and vault().location are supported"
+        },
+        "allowCustomProperties": {
+          "type": "boolean",
+          "description": "A value indicating whether the custom properties within the property bag are allowed. Needs api-version to be specified in the policy rule eg - vault('2019-06-01')."
+        }
+      },
+      "description": "The custom resource function definition."
+    },
+    "DataManifestResourceFunctionsDefinition": {
+      "properties": {
+        "standard": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The standard resource functions (subscription and/or resourceGroup)."
+        },
+        "custom": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataManifestCustomResourceFunctionDefinition"
+          },
+          "description": "An array of data manifest custom resource definition."
+        }
+      },
+      "description": "The resource functions supported by a manifest"
+    },
+    "DataEffect": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The data effect definition."
+        },
+        "detailsSchema": {
+          "type": "object",
+          "description": "The data effect details schema."
+        }
+      },
+      "description": "The data effect definition."
+    },
+    "AliasPathMetadata": {
```

Comment:
can you ref these alias* definitions from specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/resources.json